package.json

```diff
@@ -30,6 +30,8 @@
     "@wordpress/element": "2.16.0",
     "@wordpress/i18n": "3.14.0",
     "@wordpress/icons": "2.10.0",
+    "@wordpress/is-shallow-equal": "3.1.2",
```

Comment:
Is this needed? I don't see it being used anywhere. Also why do we need to include babel?

Reply:
I was getting test errors when removed, but I'll try again. I think it might be a peer dependency that was coming in on WC Components.

Reply:
Yeah, tests fail. It is an unmet dependency of the g2 components.